### ISSUE TITLE
Pretty JSON view double-escapes quotes after May 28 update

### ISSUE THREAD

**Issue by ross-p** [a85/POSTMan-Chrome-Extension#1087]
My postman packaged app just updated today May 28.  The new one looks great.

However there is a bug with the Pretty JSON response view.  It escapes quotes twice.

Sample raw JSON response:

``` json
{ "foo": "{\"bar\": true}" }
```

And here is what Postman shows in the "pretty" tab:

``` json
{
  "foo": "{\\\"bar\\\": true}"
}
```

**Comment by ross-p** [comment]
Version 3.0.0.5 fwiw

**Comment by abhijitkane** [comment]
@ross-p We'll fix this in the coming release.

**Comment by fightersmurf** [comment]
Error still present in 
Version 3.0.0.9 (packaged)
OS: win
Architecture: x86-64
Native client architecture: x86-64

**Comment by abhijitkane** [comment]
I'm not able to reproduce this now - 
Pretty view: ![screen shot 2015-06-25 at 19 49 30](https://cloud.githubusercontent.com/assets/681190/8356496/555ad794-1b73-11e5-8f7e-7b847d61f570.png)

Raw view: ![screen shot 2015-06-25 at 19 49 36](https://cloud.githubusercontent.com/assets/681190/8356497/5584bc76-1b73-11e5-9faa-dd5b4e5435b4.png)

This is on 3.0.0.9

**Comment by fightersmurf** [comment]
In my case it happens in an URL 
raw view:
![postman_2015-06-25_16-32-37](https://cloud.githubusercontent.com/assets/13050126/8356931/7218778a-1b58-11e5-8c31-db15cf9e5479.png)

Pretty-print:
![postman_2015-06-25_16-28-50](https://cloud.githubusercontent.com/assets/13050126/8356954/91a732d0-1b58-11e5-8fe6-0d764b3f76ca.png)

**Comment by ross-p** [comment]
@fightersmurf that is valid JSON.  Not sure why JSON likes to escape slashes by default, but it does.  Probably to guarantee that `</script>` doesn't appear anywhere, which would break a lot of JavaScript parsers.  (JSON was first for JavaScript after all).

EDIT: Wait, I'm not sure the slash should be escaped twice though.  It should probably be `\/` instead of `\\/`, but you probably need to open a new issue about that.

**Comment by ross-p** [comment]
@abhijitkane Confirmed fixed in 3.0.0.9, thank you!  :)

**Comment by Radu3000** [comment]
This is NOT fixed in 3.0.5 - the double quotes are still escaped :( !!!

**Comment by sgb-io** [comment]
+1, this seems to be an issue for me too, same as @fightersmurf. v `3.0.5`
